h2o-algos/src/main/java/hex/tree/SharedTreeModel.java

```diff
@@ -175,6 +175,39 @@ public SharedTreeOutput( SharedTree b) {
       _modelClassDist = _priorClassDist;
     }
 
+    @Override
+    public TwoDimTable createInputFramesInformationTable(ModelBuilder modelBuilder) {
```

Comment:
why is this defined on SharedTreeModel level?

the only tree-specific parameter is calibration frame

Reply:
will refactor so that it uses table = super.createInputFrames.. and just fills calibration frame here